Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -1149,24 +1150,24 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
           _("Moves all objects according to the forces they have. GDevelop "
             "calls this action at the end of the events by default."),
           _("Apply movement to all objects"),
-          _("Movement"),
+          _("Movement using forces"),
           "res/actions/doMove24.png",
           "res/actions/doMove.png")
       .AddCodeOnlyParameter("currentScene", "")
       .MarkAsAdvanced();
 
   extension
       .AddCondition("SeDirige",
-                    _("An object is moving toward another"),
+                    _("An object is moving toward another (using forces)"),
                     _("Check if an object moves toward another.\nThe first "
                       "object must move."),
                     _("_PARAM0_ is moving toward _PARAM1_"),
-                    _("Movement"),
+                    _("Movement using forces"),
                     "res/conditions/sedirige24.png",
                     "res/conditions/sedirige.png")
       .AddParameter("objectList", _("Object"))
       .AddParameter("objectList", _("Object 2"))
-      .AddParameter("expression", _("Angle of tolerance"))
+      .AddParameter("expression", _("Angle of tolerance in degrees"))
```

Comment:
nitpicking: let's normalise this description so that it's like to other: "Tolerance, in degrees"